.github/workflows/check_package.yml

```diff
@@ -11,13 +11,13 @@ on:
 
 jobs:
   get_package_info:
-    name: Get package info for ${{ inputs.package-name }}
+    name: Get package info
     uses: ./.github/workflows/get_package_info.yml
     with:
       package-name: ${{ inputs.package-name }}
   check_codegen:
     if: ${{ needs.get_package_info.outputs.should-check-codegen == 'true' }}
-    name: Check codegen for ${{ inputs.package-name }}
+    name: Codegen
```

Comment:
Removing the package name makes these indistinguishable on the left hand pane.

<img width="429" height="1096" alt="image" src="https://github.com/user-attachments/assets/048a821f-d560-4135-b860-b6f4e3365165" />